### PR TITLE
Always go bleeding edge

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
# Description

A minor update after I saw we don't go bleeding edge all the time.

This was tested and depends on recent pull requests on Arizona, after which we should re-run CI.

- [x] I have performed a self-review of my changes
- [x] I have read and understood the [contributing guidelines](/arizona-framework/arizona_example/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: rebar.lock
Comment (paulo-ferraz-oliveira) [inline rebar.lock:1]: 👋

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ _build
 doc/
 erl_crash.dump
 rebar3.crashdump
+rebar.lock
```
Comment (williamthome) [inline .gitignore:5]: Is this to keep the example updated with Arizona's main branch?

Comment (paulo-ferraz-oliveira) [reply .gitignore:5]: Yeah. And then we can have a workflow that runs every now and then to warn us if something's broken.

We want to showcase the latest, at all times, right? I imagine if it's in the main branch it's good to go, even if not tagged 😄

Comment (paulo-ferraz-oliveira) [reply .gitignore:5]: On the other hand, if you prefer (but that'd have to wait for a tag) we can also add Renovate to this one and it'll move alongside the tagged releases 😄

Comment (williamthome) [reply .gitignore:5]: LGTM. Let's go by the easiest way now :D

**Pull request merged**